docker/wmagent/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM cmssw/cmsweb:20221130-stable
+FROM registry.cern.ch/cmsweb/cmsweb:20221130-stable
```

Comment:
I think we have a more recent cmsweb image. I'd suggest to update it to the latest.